src/crimson/os/seastore/transaction.h

```diff
@@ -228,9 +234,9 @@ class Transaction {
     delayed_temp_offset = 0;
     read_set.clear();
     invalidate_clear_write_set();
-    fresh_block_list.clear();
     mutated_block_list.clear();
     delayed_alloc_list.clear();
+    inline_block_list.clear();
```

Comment:
I think ool_block_list also have to be cleared, am I right?

Reply:
Yep, fixed.